src/GeneralStateTestsFiller/stSystemOperationsTest/doubleSelfdestructTouchFiller.yml

```diff
@@ -0,0 +1,110 @@
+doubleSelfdestructTouch:
+  _info:
+    comment: |
+      A single contract can execute SELFDESTRUCT multiple times using by being called
+      multiple times. The second and later SELFDESTRUCTs have little effect but can
+      touch some new beneficiary addresses.
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 1
+    currentGasLimit: 30000000
+    currentNumber: 2
+    currentTimestamp: 999
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    # Tx sender
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      nonce: 0
+      balance: 100000001
+      storage: {}
+      code: ''
+
+    # Tx target contract
+    000000000000000000000000000000000000c0de:
+      nonce: 0
+      balance: 0
+      storage: {}
+      code: |
+        :yul {
+          let r1 := call(70000, 0xdead, callvalue(), 0, 0, 0, 0)
+          let r2 := call(70000, 0xdead, 0, 0, 0, 0, 0)
```

Comment:
what if you put a balance when running the second time as well?

Reply:
Ah, good point. You can increase the balanace of already selfdestructed account. This is also not covered by tests because I can see my implementations is buggy. I will add more variants then.